src/System.Runtime.Extensions/ref/System.Runtime.Extensions.cs

```diff
@@ -634,6 +634,7 @@ public static partial class Path
         public static string GetTempPath() { return default(string); }
         public static bool HasExtension(string path) { return default(bool); }
         public static bool IsPathRooted(string path) { return default(bool); }
+        public static string GetRelativePath(string relativeTo, string path) { return default(string); }
```

Comment:
How will this work? Do we need to have two "prerelease" contract versions, one for the next version of netstandard, and one for the next version of necoreapp?